.github/workflows/releaser.yml

```diff
@@ -267,4 +267,4 @@ jobs:
       with:
         files: .github/releasers/pactus_downloader.sh
       env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Ensure there is always a new line at the end of each file. Please install EditorConfig from https://editorconfig.org/.